internal/resources/resource_volume.go

```diff
@@ -100,6 +105,14 @@ func volumeSchema() map[string]*schema.Schema {
 			},
 		},
 
+		vSizeInUse: {
+			Type:        schema.TypeFloat,
+			Required:    false,
+			Optional:    false,
+			Computed:    true,
+			Description: "The amount of the volume currently used as reported by the array in GBytes.",
```

Comment:
This does not match the comment for "CapacityUsed" attribute in hpegl-metal-client (https://github.com/HewlettPackard/hpegl-metal-client/pull/128/files) -
```
// The amount of the volume currently used as reported by the array in KiB
```

Reply:
As we convert the value, had to update the comment.
Please refer the the reply on the below comment.

Reply:
Ok thanks. Keeping vSize and vSizeInUse units the same is good.